code/game/objects/effects/particles_system.dm

```diff
@@ -373,3 +378,62 @@ var/list/particle_string_to_type = list(
 	icon_state = "risen"
 
 	plane = ABOVE_HUD_PLANE
+
+//ZAS DUST
+/particles/zas_dust
+	width = 96
+	height = 96
+	count = 20
+	spawning = 2
+
+	color = "#FFFFFF99"
+	lifespan = 1 SECONDS
+	fade = 0.5 SECONDS
+	icon = 'icons/effects/effects_particles.dmi'
+	icon_state = "zas_dust"
+	position = generator("box", list(-15,-15), list(15,15))
+	velocity = list(0,0)
+
+/turf
+	var/last_dust_time = 0
+	var/last_dust_strength = 0
+
+/turf/proc/flying_dust(var/turf/dest, var/wind_strength = 3, var/wind_opacity = 128)
+	if (last_dust_time == SSair.times_fired && last_dust_strength > wind_strength)
+		return
+	last_dust_time = SSair.times_fired
+	last_dust_strength = wind_strength
+	var/this_dust_time = last_dust_time
+	add_particles(PS_ZAS_DUST)
+	adjust_particles(PVAR_SPAWNING, 2, PS_ZAS_DUST)
```

Comment:
is there a reason you're adjusting this here aswell?

also, i think 0.5 looks better

Reply:
yes there is, since the dust is finer, it looks better with a bit more of it. Also This was the original value I went with, then toned down when I originally made the PR. I think this value works better with the new dust sprite.

Reply:
no i mean why you're adjusting spawning to 2 again, the default is 2 (L387)
something like this would make sense if each turf had a dustiness var which would then determine how much dust it spawns

Reply:
> no i mean why you're adjusting spawning to 2 again

Because when the particle turns off it does so by just setting spawning to 0. This is better than qdel'ing/re-creating the particle holder each time.